### ISSUE TITLE
Train PPT but got Loss = nan

### ISSUE THREAD

**Issue by isunLt** [Pointcept/Pointcept#257]
Thanks for sharing the code! 
We follow the readme to prepare the enviorment and run the command: 
``sh scripts/train.sh -g 4 -d nuscenes -c semseg-ppt-v1m1-0-nu-sk-wa-spunet -n semseg-ppt-v1m1-0-nu-sk-wa-spunet``
However, the loss turn to Nan in the middle of the training. Could the authors kindly help me with this problem?
<img width="938" alt="8a8aa6e13c926363652fb225a3a36de" src="https://github.com/Pointcept/Pointcept/assets/37700210/9ef00b7b-8a56-45bf-b27f-7adc4d438351">

**Comment by Gofinge** [comment]
Try again or set amp False. It does occur from time to time. Maybe clip loss within a range can solve it but I have not try it.

**Comment by isunLt** [comment]
> Try again or set amp False. It does occur from time to time. Maybe clip loss within a range can solve it but I have not try it.

Thank you for your timely responce. I have tried twice but all get loss = nan during the training. I will try to set amp False.

**Comment by isunLt** [comment]
> Try again or set amp False. It does occur from time to time. Maybe clip loss within a range can solve it but I have not try it.

I have tried `semseg-ppt-v1m2-0-nu-sk-wa-spunet` to use the decoupled segmentation head to train the network. It successfully runs through the training procedure but fails during the testing procedure as shown in the Fig. Can you help me with this problem so that I can successfully run the testing procedure?

![微信截图_20240719203933](https://github.com/user-attachments/assets/659b960f-fbb4-4f99-a1e3-4b47301e2de4)
